api/src/main/java/net/md_5/bungee/api/event/ChatEvent.java

```diff
@@ -12,10 +13,18 @@
 @Data
 @ToString(callSuper = true)
 @EqualsAndHashCode(callSuper = true)
-public class ChatEvent extends TargetedEvent implements Cancellable
+public class ChatEvent extends AsyncEvent<ChatEvent> implements Cancellable
```

Comment:
Changing the type of an event without mentioning in the pull request description is not that good.